docs/kook.md

```diff
@@ -0,0 +1,25 @@
+# Integrating the KOOK Provider
+
+## Example
+
+```csharp
+services.AddAuthentication(options => /* Auth configuration */)
+        .AddKook(options =>
+        {
+            options.ClientId = "my-client-id";
+            options.ClientSecret = "my-client-secret";
+
+            // The scope get_user_info is added by default
+            // Please make sure the scope get_user_info is enabled in your KOOK developer center
+            // Hence, whether to add the scope get_user_info here manually is optional
+            // options.Scope.Add("get_user_info");
```

Comment:
If it's added by default in the provider, then adding it manually would be a no-op.

I would suggest that the commented out line of code would instead _remove_ it if a user didn't want to enable it in the KOOK developer centre.

Reply:
Thanks for reviewing.
Did you mean that the provider should not add a scope that can be disabled by developers in the KOOK developer center by default?

Reply:
Add a default scope is fine if that's what the average user would do with the provider, but the comment should demonstrate removing it in the case someone doesn't wish to use it.

It's just that the comment doesn't make much sense adding something that's already there.

Reply:
Got it!